tests/test_md.py

```diff
@@ -39,35 +39,29 @@ def test_multiple_steps_store_interval(self):
         bps = [bp]
 
         ctxt = custom_ops.Context(x0, v0, box, intg, bps)
-        test_du_dls, test_xs, test_boxes = ctxt.multiple_steps(np.zeros(10), 10, 10)
+        test_xs, test_boxes = ctxt.multiple_steps(10, 10)
         assert len(test_xs) == 1
-        assert len(test_du_dls) == 1
         assert len(test_xs) == len(test_boxes)
         # We should not get out the input frame
         assert np.any(np.not_equal(x0, test_xs[0]))
 
         # The current coordinates should match, as the number of steps and the interval match
         np.testing.assert_array_equal(test_xs[0], ctxt.get_x_t())
-        _, test_frame_du_dl, _ = bps[0].execute(test_xs[0], test_boxes[0], 0.0)
-        np.testing.assert_array_equal(test_du_dls[0], test_frame_du_dl)
+        _, _ = bps[0].execute(test_xs[0], test_boxes[0])
 
         # Given an interval greater than the number of steps, return empty arrays
-        test_du_dls, test_xs, test_boxes = ctxt.multiple_steps(np.zeros(10), 100, 100)
+        test_xs, test_boxes = ctxt.multiple_steps(10, 100)
         assert len(test_xs) == 0
-        assert len(test_du_dls) == 0
         assert len(test_boxes) == 0
 
         # Given interval of 0, return the last frame
-        test_du_dls, test_xs, test_boxes = ctxt.multiple_steps(np.zeros(10), 0, 0)
+        test_xs, test_boxes = ctxt.multiple_steps(10, 0)
         assert len(test_xs) == 1
-        assert len(test_du_dls) == 1
         assert len(test_boxes) == 1
 
         np.testing.assert_array_equal(test_xs[0], ctxt.get_x_t())
-        _, test_frame_du_dl, _ = bps[0].execute(test_xs[0], test_boxes[0], 0.0)
-        np.testing.assert_array_equal(test_du_dls[0], test_frame_du_dl)
+        _, _ = bps[0].execute(test_xs[0], test_boxes[0])
 
-    @pytest.mark.skip("multiple_steps_U is deprecated")
```

Comment:
This was mistakenly skipped in #893. Re-enable and update for the new `multiple_steps_U` interface.